src/App.js

```diff
@@ -12,8 +14,10 @@ function App() {
       </header>
       <Nav />
       <Routes>
-        <Route path="/" element={<Articles />}></Route>
+        <Route path="/articles" element={<Articles />}></Route>
```

Comment:
This route isn't self closing but the other routes are. ALSO, why not just make this page the default '/' path as we discussed yesterday?